### PR TITLE
Switching HeaderFooterGridView for StaggeredGridView

### PR DESCRIPTION
**Pull request by dcluna**
To avoid the bug in which the 'Logged In:' text gets cut off.

### CODE REVIEW

File: app/AndroidManifest.xml [hunk 1]
```diff
@@ -426,7 +426,7 @@
         android:name="android.permission.USE_CREDENTIALS" />
 
     <uses-sdk
-        android:minSdkVersion="8"
+        android:minSdkVersion="10"
```
Comment (dcluna) [inline app/AndroidManifest.xml]: this is required by staggeredgridview

File: build.gradle [hunk 2]
```diff
@@ -65,6 +66,11 @@ project(':libraries:mapballoons') {
   }
 }
 
+ext {
+  compileSdkVersion = "Google Inc.:Google APIs:21"
+  buildToolsVersion = "21.1.2"
```
Comment (phillipm) [inline build.gradle]: This needs to be 22

File: build.gradle [hunk 3]
```diff
@@ -86,8 +92,8 @@ android {
     }
 
     defaultConfig {
-        minSdkVersion 9
-        targetSdkVersion 22
+        minSdkVersion 10
+        targetSdkVersion 21
```
Comment (phillipm) [inline build.gradle]: 22

File: app/AndroidManifest.xml (hunk 1 above)
Comment (ctsims) [inline app/AndroidManifest.xml]: We can't change this, unfortunately, our app needs to be able to target SDK 8, and last time we changed this a few projects got blocked.

Comment (dcluna) [inline app/AndroidManifest.xml]: Then we cannot merge this PR. I'll search for other alternatives.

**Comment by dcluna** [comment]
Closed because we need to support sdk version 8

**Pull request closed without merging**